src/Symfony/Component/Serializer/Mapping/AttributeMetadataInterface.php

```diff
@@ -53,14 +53,33 @@ public function getMaxDepth();
 
     /**
      * Sets the serialization name for this attribute.
+     *
+     * @deprecated
```

Comment:
Please add more information (like we do on other @deprecated tags)

Reply:
done.